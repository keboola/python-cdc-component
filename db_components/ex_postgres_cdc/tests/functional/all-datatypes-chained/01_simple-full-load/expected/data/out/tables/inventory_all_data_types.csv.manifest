{"columns": ["id", "col_smallint", "col_integer", "col_int", "col_bigint", "col_decimal", "col_numeric", "col_real", "col_double_precision", "col_smallserial", "col_serial", "col_bigserial", "col_money", "col_character", "col_char", "col_character_varying", "col_varchar", "col_text", "col_bytea", "col_timestamp", "col_timestamp_with_time_zone", "col_date", "col_time", "col_time_with_time_zone", "col_interval", "col_boolean", "col_point", "col_cidr", "col_inet", "col_macaddr", "col_macaddr8", "col_bit", "col_bit_varying", "col_uuid", "col_xml", "col_json", "col_jsonb", "col_integer_array", "col_int4range", "KBC__OPERATION", "KBC__EVENT_TIMESTAMP_MS", "KBC__DELETED", "KBC__BATCH_EVENT_ORDER"], "enclosure": "\"", "delimiter": ",", "metadata": [], "column_metadata": {"id": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}, {"key": "KBC.datatype.length", "value": 10}], "col_smallint": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 5}], "col_integer": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_int": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_bigint": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 19}], "col_decimal": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "NUMERIC"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 5}], "col_numeric": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "NUMERIC"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 5}], "col_real": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "NUMERIC"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 8}], "col_double_precision": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "NUMERIC"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 17}], "col_smallserial": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}, {"key": "KBC.datatype.length", "value": 5}], "col_serial": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}, {"key": "KBC.datatype.length", "value": 10}], "col_bigserial": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}, {"key": "KBC.datatype.length", "value": 19}], "col_money": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "NUMERIC"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_character": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_char": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_character_varying": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_varchar": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_text": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 16777216}], "col_bytea": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_timestamp": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "TIMESTAMP"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 29}], "col_timestamp_with_time_zone": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "TIMESTAMP"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 35}], "col_date": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "DATE"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 13}], "col_time": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "TIMESTAMP"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 15}], "col_time_with_time_zone": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "TIMESTAMP"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 21}], "col_interval": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 49}], "col_boolean": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "BOOLEAN"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 1}], "col_point": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_cidr": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_inet": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_macaddr": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_macaddr8": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_bit": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_bit_varying": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_uuid": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_xml": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_json": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_jsonb": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "col_integer_array": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 10}], "col_int4range": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": true}, {"key": "KBC.datatype.length", "value": 2147483647}], "KBC__OPERATION": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "STRING"}, {"key": "KBC.datatype.nullable", "value": false}], "KBC__EVENT_TIMESTAMP_MS": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "TIMESTAMP"}, {"key": "KBC.datatype.nullable", "value": false}], "KBC__DELETED": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "BOOLEAN"}, {"key": "KBC.datatype.nullable", "value": false}], "KBC__BATCH_EVENT_ORDER": [{"key": "KBC.description", "value": ""}, {"key": "KBC.datatype.basetype", "value": "INTEGER"}, {"key": "KBC.datatype.nullable", "value": false}]}, "write_always": false}