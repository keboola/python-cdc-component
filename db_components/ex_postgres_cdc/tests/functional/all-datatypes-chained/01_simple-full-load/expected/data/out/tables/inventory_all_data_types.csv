1,1,2,3,4,5.00,6.00,7.0,8.0,9,10,11,12.0,a         ,b         ,c,d,e,3q2+7w==,2004-10-19 10:23:54,2004-10-19 08:23:54+00,2004-10-19,37434000,08:23:54Z,37091106000000,true,"{""x"":1.0,""y"":1.0,""wkb"":""AQEAAAAAAAAAAADwPwAAAAAAAPA/"",""srid"":null}",192.168.1.0/24,192.168.1.1,08:00:2b:01:02:03,08:00:2b:01:02:03:04:05,gAI=,gAI=,a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11,<foo>bar</foo>,"{""key"": ""value""}","{""key"": ""value""}","[1,2,3]","[1,10)",r,false,0
2,2,3,4,5,6.00,7.00,8.0,9.0,10,11,12,13.0,b         ,c         ,d,e,f,3q2+7w==,2005-11-20 11:24:55,2005-11-20 08:24:55+00,2005-11-20,41095000,08:24:55Z,71368567000000,false,"{""x"":2.0,""y"":2.0,""wkb"":""AQEAAAAAAAAAAAAAQAAAAAAAAABA"",""srid"":null}",192.168.2.0/24,192.168.2.2,08:00:2b:02:03:04,08:00:2b:02:03:04:05:06,AAM=,AAM=,b1ffbc99-9c0b-4ef8-bb6d-6bb9bd380a12,<bar>foo</bar>,"{""key1"": ""value1""}","{""key1"": ""value1""}","[2,3,4]","[2,11)",r,false,1
